debian/system76-driver.postinst

```diff
@@ -21,6 +21,14 @@
 # with `system76-driver`; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+lsb_description="$(sed -n -e 4p /etc/lsb-release)"
```

Comment:
I think `grep DESCRIPTION /etc/lsb-release` would be more reliable, in case they decide to change the ordering later down the road.

Reply:
I also think that maybe we should do the reverse? Rather than checking if it is `Ubuntu`, we could check if it is not `Pop!`, so Xubuntu / elementary / etc could use it, too.

Reply:
Spoke with @djordan2 thought that this is a fine fix for now and that we probably want to include in documentation for the extension and system76 power that using one will probably want the other (and that you'll probably need to enable the extension yourself).